Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -16,71 +16,85 @@
+
+When dealing with a particularly complicated bug (one that would require a rather significant effort to address) the release engineer should speak with either Andreea Papillon (Native Apps' Product Manager) or Lilla Szulyovszky (Native Apps' Delivery Manager) as every release is managed and signed-off by the Native Apps squad.  
 The bug's corresponding feature should be disabled altogether using its feature switch (if applicable). Such a bug would subsequently be handled by its respective squad.
 
 Release duties have priority over regular squad duties. Please inform your squad of your unavailability before starting to work in the release. As with every other week, twenty percent of your work hours overall are to be allocated towards making the release a reality.
 
 There are usually two release engineers working at any given time. It goes without saying that both engineers need to work together and that constant feedback is vital.
 
-### 2. Release step-by-step
-
-**Phase 1: Initiation**
-<br/>	*It starts at the end of the sprint (typically when the new sprint starts on Monday)*
-
-1. Cut a release branch:  Create a new branch from develop and push to origin (e.g `release/3.2.0`).
-1. Create a slack channel to discuss anything relative to the release (e.g. `ios_release_3_2_0`).
-1. Bump the release version by triggering its command (eg. `/release babylon:3.2.0`) in `#ios-build` (you can run the command every time you want to upload a new build).
-    * This creates a test Tesflight build (try to make one as early as possible so that you can catch issues like missing/expired certificates or profiles and any other production build errors early).
-1. Trigger a hockey build from that branch using its command (eg. `/hockeyapp Babylon branch:release/3.17.0`) in `#ios-build`.
-1. Create and share the changelog in `#ios-launchpad`: In console run `git log --since="2018-11-26" --pretty=format:%s  > issue_list.txt` (use the date when the sprint started). Ask the channel for the expected release notes from each squad if they are releasing anything.
+## 2. Release step-by-step
+
+### Phase 1: Initiation
+*It starts at the end of the sprint (typically when the new sprint starts on Monday)*
+
+1. Cut a release branch
+   * Create a new branch from `develop`, and be sure to name it using the `release/{appname}/{version}` convention (e.g. `release/babylon/4.1.0`) – where `appname` is one of `babylon`, `telus`, `nhs111` or `bupa`
+   * Push the branch to `origin`
+1. Create a slack channel to discuss anything relative to the release (e.g. `ios_release_4_1_0`).
+1. Bump the release version by triggering the Slack command (eg. `/release babylon:4.1.0`) in `#ios-build` (you can run the command every time you want to upload a new build).
```

Comment:
ℹ️ This currently needs [a fix in our Steve bot so that it takes the new branch naming convention into account](https://github.com/Babylonpartners/ios-build-distribution/blob/master/Sources/AppCore/Models/Constants.swift#L47).

I'll do a PR in `ios-build-distribute` for this soon.